### DISCUSSION TITLE
Support for ESLint 9 in @tanstack/eslint-plugin-query

### DISCUSSION THREAD

**Discussion by waiyan13** [TanStack/query#7250] [Ideas]
[ESLint 9](https://eslint.org/blog/2024/04/eslint-v9.0.0-released/) has been released.

Any timeline estimate on supporting the new version in future releases of the plugin?

**Comment by TkDodo** [comment]
good question. what would it take from our end to support it? what errors are you running into?

**Reply by waiyan13** [reply]
I haven't run extensively yet. Currently, it is just conflicting peer dependency issue when upgrading.

I opened this thread hoping others will report in their respective issues.

Also, thank you for all the work you have done with Tanstack query.

**Comment by lucsky** [comment]
See #6669
